data/scenarios/02Tutorials/09-crash.yaml

```diff
@@ -0,0 +1,111 @@
+name: Debug robots
+description: |
+  Learn how to view built robots and debug them.
+goal:
+  - Before you send your robots far away you need to learn how
+    to figure out what went wrong with them if they crash.
+  - Send a robot to walk four steps straight east into the mountain,
+    but use the bind syntax 'r <- build {COMMANDS}'.
+  - |
+    Since you expect the robot will crash into the mountain,
+    add a 'log' command so that we will see what command failed:
+  - |
+    'r <- build {log "Hi!"; move;move;move; log "3"; move; log "OK"}'
+  - After that you can 'view r', to see how far it got.
+  - The instructions on how to win will appear in the crashed robots log.
+entities:
+  - name: Win
+    display:
+      attr: device
+      char: 'W'
+    description:
+      - Do 'place "Win"' once you are done with this challenge.
+      - You might need to 'grab' the entity you are standing on
+        or move to an empty cell.
+    properties: [known, portable]
+  - name: knownwater
+    display:
+      attr: water
+      char: '~'
+    description:
+    - A running stream of of water.
+    properties: [known, portable, growable, liquid]
+    growth: [0,0]
+  - name: knowntree
+    display:
+      attr: plant
+      char: 'T'
+    description:
+    - A tree.
+    properties: [known, portable, growable]
+    growth: [0,0]
+  - name: knownmountain
+    display:
+      attr: snow
+      char: 'A'
+    description:
+    - A mountain.
+    properties: [known, unwalkable]
+    growth: [0,0]
+win: |
+  try {
+    as base {has "Win"}
+  } { return false }
+solution: |
+  crasher <- build {move; move; move; log "bye"; move};
+  wait 1000;
```

Comment:
I meant, because the test timeout, perhaps a lower value would make the win condition works better? In case the test instance is overloaded or something... Also it seems like the test timeout after 1 second by default, so maybe pushing that to 2 second could help too?

Reply:
You know it does actually take 0.44s to run the test, unlike the others which take ~0.10s.

Reply:
I pushed to and restarted #465, let us see... :eyes: